addons/account_edi_ubl_cii/models/account_edi_xml_ubl_20.py

```diff
@@ -701,6 +703,13 @@ def _get_document_type_code_vals(self, invoice, invoice_data):
     # IMPORT
     # -------------------------------------------------------------------------
 
+    def _import_invoice_ubl_cii(self, invoice, file_data, new=False):
+        """ Allow AttachedDocument files to have their original record properly parsed. """
+        if file_data['xml_tree'].tag.split('}')[-1] == 'AttachedDocument':
```

Comment:
```suggestion
        if etree.QName(file_data['xml_tree']).localname == 'AttachedDocument':
```
should normally work according to the LXML docs

https://lxml.de/3.7/tutorial.html#:~:text=tag%20%3D%20etree.QName(root)